package.json

```diff
@@ -73,8 +73,8 @@
     "@celo/wallet-rpc": "~1.2.0",
     "@coinbase/cbpay-js": "^1.0.2",
     "@crowdin/ota-client": "^0.4.0",
-    "@fiatconnect/fiatconnect-sdk": "^0.4.0",
-    "@fiatconnect/fiatconnect-types": "^8.0.0",
+    "@fiatconnect/fiatconnect-sdk": "^0.5.0",
+    "@fiatconnect/fiatconnect-types": "^10.1.0",
```

Comment:
is this change necessary for "upgrade @react-navigation/drawer to v6 and enable freeze"? if not can we please remove related changes from this PR and open a separate one?